app/assets/javascripts/discourse/app/components/user-status-message.hbs

```diff
@@ -6,7 +6,7 @@
     </span>
   {{/if}}
   {{#if this.showTooltip}}
-    <DTooltip>
+    <DTooltip @onTrigger={{@onTrigger}} @onUntrigger={{@onUntrigger}}>
```

Comment:
I saw that you used the onTrigger event in https://github.com/discourse/discourse-assign/pull/457 to call `event.stopPropagation()`.

But what if we call `event.stopPropagation` right here in the `UserStatusMessage` component? That way we won't need to expose the `onTrigger` and `onUntrigger` events (so API of the `UserStatusMessage` will be simpler), and we won't need to handle it every time we need it.

Reply:
Maybe we can even encapsulate it in the `DTooltip` component?

Reply:
Yes, I agree. It might be better to include `event.stopPropagation` inside `DTooltip`. I don't know how useful it would be, but we'll see. It was originally intended just for when the `click` event was used.